docs/network/interfaces_and_networks.md

```diff
@@ -47,14 +47,15 @@ fields:
 </tr>
 <tr class="even">
 <td><p><code>multus</code></p></td>
-<td><p>Secondary network provided using Multus</p></td>
+<td><p>Secondary network provided using Multus<br>
+or Primary network when Multus is defined as default</p></td>
 </tr>
 </tbody>
 </table>
 
 ### pod
```

Comment:
nit:
```suggestion
### Pod
```

Reply:
not sure about this, I want to be consistent with the exact value of the schema

Reply:
Not a must.
Please resolve the thread.